test/test-meshctl-status.js

```diff
@@ -36,8 +36,8 @@ test('Test status command', function(t) {
     'Environment variables:',
     '  No environment variables defined',
     'Instances:',
-    '    Version  Agent version  Cluster size',
-    '     1.2.3       7.8.9            3',
+    '    Version  Agent version  Cluster size  Driver metadata',
+    '     1.2.3       7.8.9            3              {}',
```

Comment:
It will print `N/A` if there is no metadata, the mock instance used for the tests just happens to have an empty object for its metadata.